requirements/requirements-essential.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.11.21
```

Comment:
The tests are now passing with the changes you made, just one thing, please remove this change because the Django version is being set by Github Actions.